app/delivery/send_to_providers.py

```diff
@@ -165,6 +166,25 @@ def check_for_malware_errors(document_download_response_code, notification):
         document_download_internal_error(notification=notification)
 
 
+def check_service_over_bounce_rate(service_id: str):
+    if not current_app.config["FF_BOUNCE_RATE_V1"]:
+        return
+
+    bounce_rate = bounce_rate_client.get_bounce_rate(service_id)
+    bounce_rate_status = bounce_rate_client.check_bounce_rate_status(service_id)
+    debug_data = bounce_rate_client.get_debug_data(service_id)
+    current_app.logger.debug(f"Bounce Rate: {bounce_rate} Bounce Status: {bounce_rate_status}, Debug Data: {debug_data}")
```

Comment:
Change as per @smcmurtry in slack:
```suggestion
    current_app.logger.info(f"Service id: {service_id} Bounce Rate: {bounce_rate} Bounce Status: {bounce_rate_status}, Debug Data: {debug_data}")
```